Documentation/other_apis.md

```diff
@@ -1,10 +1,36 @@
 # Members API
 
+* [Getting the etcd version](#getting-the-etcd-version)
```

Comment:
You're right.  I meant to put it above.  Ill update.
On Dec 15, 2015 4:35 PM, "Xiang Li" <notifications@github.com> wrote:

> In Documentation/other_apis.md
> <https://github.com/coreos/etcd/pull/4001#discussion_r47721647>:
>
> > @@ -1,10 +1,36 @@
> >  # Members API
> >
> > +* [Getting the etcd version](#getting-the-etcd-version)
>
> these two are not part of Members API. Move it out from # Members API?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/4001/files#r47721647>.
>